examples/redeploy-script/redeploy.sh

```diff
@@ -155,6 +155,10 @@ if [[ $TRAIN_RESULT -ne 0 ]]; then
 fi
 
 # Deploy
+# Get current running instance PID
+PIDBYPORT_COMMAND="lsof -t -i:$PORT"
+DEPLOYEDPORT=$($PIDBYPORT_COMMAND)
```

Comment:
Instead of using `DEPLOYEDPORT`, I think maybe `DEPLOYEDPID` is more accurate?

Reply:
You are right @Wei-1 . That would be more accurate in context. Updated.